### PR TITLE
Regression test

### PR DESCRIPTION
**Pull request by AdrianDAlessandro**
This PR adds a regression test that can be run with `pytest`

The data used for this test is the Open Access publication describing this software: https://doi.org/10.3389/fdgth.2022.788124 - it is Creative Commons licensed and the html used for the test includes the Creative Commons license text inside it, which automatically makes it comply with the license, as far as I can tell.

### CODE REVIEW

File: tests/test_regression.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    """A regression test for the main autoCORPus class on the AutoCORPus Paper.
+
+    The test data output is built using `run_app.py` with the following arguments:
+    ```
+    python run_app.py -c "configs/config_pmc.json" -t "tests/data" -f tests/data/PMC8885717.html -o JSON
+    ```
+    """
+    from src.autoCORPus import autoCORPus
+
+    with open(Path(__file__).parent / "data" / "PMC8885717_abbreviations.json") as f:
```
Comment (Thomas-Rowlands) [inline tests/test_regression.py:15]: encoding="utf-8" needed to resolve Windows 11 encoding error

File: tests/test_regression.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        expected_abbreviations = json.load(f)
+    with open(Path(__file__).parent / "data" / "PMC8885717_bioc.json") as f:
+        expected_bioc = json.load(f)
+    with open(Path(__file__).parent / "data" / "PMC8885717_tables.json") as f:
```
Comment (Thomas-Rowlands) [inline tests/test_regression.py:19]: encoding="utf-8" needed to resolve Windows 11 encoding error

**Review by Thomas-Rowlands — CHANGES_REQUESTED** [review]
Added suggested changes to the test_regression.py file. Testing in Windows 11 throws an encoding error, which is resolved by specifying UTF-8. Looks good with this change however.

**Comment by AdrianDAlessandro** [comment]
@Thomas-Rowlands I've updated it to specify utf-8 encoding. Can you approve this PR to run workflows so that we can see if the test is passing?

**Review by Thomas-Rowlands — APPROVED** [review]

**Pull request merged**